x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/time_shift.tsx

```diff
@@ -78,17 +76,15 @@ export function TimeShift({
   columnId: string;
   layer: IndexPatternLayer;
   updateLayer: (newLayer: IndexPatternLayer) => void;
-  isFocused: boolean;
   activeData: IndexPatternDimensionEditorProps['activeData'];
   layerId: string;
 }) {
-  const focusSetRef = useRef(false);
   const [localValue, setLocalValue] = useState(selectedColumn.timeShift);
   useEffect(() => {
     setLocalValue(selectedColumn.timeShift);
   }, [selectedColumn.timeShift]);
   const selectedOperation = operationDefinitionMap[selectedColumn.operationType];
-  if (!selectedOperation.shiftable || selectedColumn.timeShift === undefined) {
```

Comment:
I think I understand: you removed this because we now show the multiselect all the time. Because of this, we now allow it to be empty if there's no timeshift.

Reply:
Yes exactly!